### PR TITLE
Add user namespaces support for `pause`

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
We now add support for UID and GID mappings which are required for unsharing user namespaces in `conmonrs pause`.

#### Which issue(s) this PR fixes:

Fixes https://github.com/containers/conmon-rs/issues/1043
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
Added support for unsharing user namespaces in `conmonrs pause`
```

### CODE REVIEW

File: conmon-rs/server/src/config.rs [hunk 1]
```diff
@@ -166,6 +166,26 @@ pub enum Commands {
...
+        )]
+        /// User ID mappings for unsahring the user namespace.
+        /// Allows multiple mappings in the format: "CONTAINER_ID HOST_ID SIZE".
+        uid_mappings: Vec<String>,
+
+        #[clap(
+            long("gid-mappings"),
+            required_if_eq("user", "true"),
+            short('g'),
+            value_name("MAPPINGS")
```
Comment (rphillips) [inline conmon-rs/server/src/config.rs:184]: This value_name is the same as line 174.

Comment (saschagrunert) [reply conmon-rs/server/src/config.rs]: Yeah I guess we can just remove that and use the default. Fixed as suggested.

**Comment by saschagrunert** [comment]
@rphillips @haircommander PTAL (again)

**Comment by rphillips** [comment]
lgtm

**Pull request merged**